README.md

```diff
@@ -46,6 +46,20 @@ Extend @ni/eslint-config/typescript in the ESLint configuration. Configure the @
 }
 ```
 
+## Recommended Development Environment Configuration
+Modern IDEs can be configured to provide live feedback about ESLint errors.
+
+**Visual Studio Code**
+
+Install the [ESLint Extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).
+
+![VSCode Extension](./resources/VSCodeESLintExtension.png)
```

Comment:
To make this image work from all locations it should use a githubusercontent url as described in the following PR: https://github.com/ni/javascript-styleguide/pull/22#discussion_r603324613

Reply:
Updated and added a comment